docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Expo Updates
+sidebar_title: Expo Updates
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for Expo Updates, a protocol for delivering updates to apps running on multiple platforms.
```

Comment:
What should be the name of the protocol. Is it "Expo Updates", "Expo Update" or something else entirely?

Reply:
I'd say "Expo Updates Protocol"